config/crd/bases/infrastructure.cluster.x-k8s.io_cloudstackaffinitygroups.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
TBD: since these are apparently generated, should these even be in the repo...?

Reply:
I've tried to remove these before, but heard screams from the EKS-A team.

In my opinion, they're doing it wrong and should generate these when they pull CAPC in or just use the release manifests as we post in our releases. But, last I heard they rely on these being there.

Might be nice to touch base with them on this again and potentially remove these.